load/load_linux.go

```diff
@@ -16,6 +18,29 @@ func Avg() (*AvgStat, error) {
 }
 
 func AvgWithContext(ctx context.Context) (*AvgStat, error) {
+	if runtime.GOOS == "android" {
+		return sysinfoAvgWithContext(ctx)
+	} else {
+		return fileAvgWithContext(ctx)
+	}
+}
+
+func sysinfoAvgWithContext(ctx context.Context) (*AvgStat, error) {
+	var info syscall.Sysinfo_t
+	err := syscall.Sysinfo(&info)
```

Comment:
I used `syscall` package here but the same method and struct are also in the `unix` package. Not sure which one is better to use.. The implementation looks the same so I guess it doesn't matter.